mergify_engine/actions/review.py

```diff
@@ -39,13 +40,42 @@ class ReviewAction(actions.Action):
         voluptuous.Required("message", default=None): voluptuous.Any(
             types.Jinja2, None
         ),
+        voluptuous.Required("review_bot_account", default=None): voluptuous.Any(
+            None, types.GitHubLogin
+        ),
     }
 
     silent_report = True
 
     def run(self, ctxt: context.Context, rule: rules.EvaluatedRule) -> check_api.Result:
         payload = {"event": self.config["type"]}
 
+        if self.config["review_bot_account"] and not ctxt.subscription.has_feature(
+            subscription.Features.REVIEW_BOT_ACCOUNT
+        ):
+            return check_api.Result(
+                check_api.Conclusion.ACTION_REQUIRED,
+                "Reviews with `review_bot_account` set are disabled",
+                ctxt.subscription.missing_feature_reason(
+                    ctxt.pull["base"]["repo"]["owner"]["login"]
+                ),
+            )
+
+        if self.config["review_bot_account"]:
+            permission = ctxt.client.item(
+                f"{ctxt.base_url}/collaborators/{self.config['review_bot_account']}/permission"
+            )["permission"]
+            if permission not in ("write", "maintain"):
+                return check_api.Result(
+                    check_api.Conclusion.ACTION_REQUIRED,
+                    (
+                        f"`{self.config['review_bot_account']}` account used as "
+                        "`review_bot_account` must have `write` or `maintain` permission, "
+                        f"not `{permission}`"
+                    ),
+                    "",
+                )
```

Comment:
I don't think that's a requirement. If I'm not mistaken a read-only user could post a review, it's just that it as less weight. And also `admin` could do it there.
cc @sileht for confirmation